hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OzoneAcl.java

```diff
@@ -75,9 +77,20 @@ public OzoneAcl(ACLIdentityType type, String name, ACLType acl) {
    * @param name - Name of user
    * @param acls - Rights
    */
-  public OzoneAcl(ACLIdentityType type, String name, List<ACLType> acls) {
+  public OzoneAcl(ACLIdentityType type, String name, BitSet acls) {
+    Objects.requireNonNull(type);
+    Objects.requireNonNull(acls);
```

Comment:
should we require the name to be nonnull as well?

Reply:
name might be null when type is WORLD or ANONYMOUS.

Reply:
No, I don't think so. If it is world, it has to be world. For Anonymous, you can insist that the client set the right value. So S3gateway will have to do the right thing to talk to us.